src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -41,44 +41,30 @@ class RA_GUIBASE_API KeyMappingManager
                                                    int key,
                                                    bool wheel = false );
 
+    /// Return the context index corresponding to contextName
+    /// \param contextName the name of the context
+    /// \return an invalid context if contextName has not been created (i.e. context,isInvalid())
+    Context getContext( const std::string& contextName );
```

Comment:
in multiline documentation, prefer `/** ... */` over `///`

Reply:
Well, the file is like that, and I prefer multiple /// since its easier to comment out for testing purpose

Reply:
`#if 0 ... code to comment out for testing purpose ...#endif `
As easy and quick than 
`/* ... code to comment out for testing purpose ... */` 
:p